ddtrace/tracer/sampler.go

```diff
@@ -24,6 +24,36 @@ import (
 	"golang.org/x/time/rate"
 )
 
+// samplerName specifies the identification of a sampler which was
+// responsible for a certain sampling decision.
+type samplerName int8
+
+const (
+	// samplerNone specifies that the sampling decision was already made
+	// and it's noop except propagating the decision.
+	samplerNone samplerName = math.MinInt8
+	// samplerUnknown specifies that the span was sampled
+	// but, the tracer was unable to identify the sampler.
```

Comment:
How is this possible? In what scenarios?

Reply:
This is not used in Go tracer. This is reserved value for other tracers in case it's not able to find out who triggered the sampling decision. I can remove it to avoid confusions. Wdyt?

Reply:
It looks like it's never used in the code, but it is in the RFC. I'm OK keeping it here for completeness. The RFC document doesn't say anything about when it can happen...